site_scons/site_tools/listModules.py

```diff
@@ -0,0 +1,67 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2010-2024 NV Access Limited, James Teh, Michael Curran, Peter Vágner, Joseph Lee,
+# Reef Turner, Babbage B.V., Leonard de Ruijter, Łukasz Golonka, Accessolutions, Julien Cochuyt,
+# Cyrille Bougot, Gerald Hartig
```

Comment:
NV Access employees generally don't put themselves in copyright headers any more

```suggestion
# Copyright (C) 2024 NV Access Limited
```